README.md

```diff
@@ -117,6 +117,15 @@ The supported network devices include:
 }
 ```
 
+### PCI Device
+
+You can choose to attach a pci device during the boot time of Dragonball.
+```
+./dbs-cli create --log-file $LOG_FILE --log-level $LOG_LEVEL --kernel-path $KERNEL_PATH --rootfs $ROOTFS_PATH --boot-args "console=ttyS0 tty0 reboot=k debug panic=1 root=/dev/vda1" --hostdev-id $HOST_DEVICE_ID --sysfs-path $SYSFS_PATH --bus-slot-func $BUS_SLOT_FUNC --vendor-device-id $VENDOR_DEVICE_ID
```

Comment:
Could you give an example of how to acquire those values? For example:

Get Xxxxxx's `hostdev-id` by `lspci`

```bash
$ lspci
02:00.0 Xxxxxx
$ HOST_DEVICE_ID="02:00.0"
```

I think leaving the part of adding PCI device is enough. Otherwise, it looks too complicated to read.

```
--hostdev-id $HOST_DEVICE_ID \
--sysfs-path $SYSFS_PATH \
--bus-slot-func $BUS_SLOT_FUNC \
--vendor-device-id $VENDOR_DEVICE_ID
```